deployment/docker/kafka-puller/run_app.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+bootstrap_server="$1"
+topic_name="$2"
+./gradlew build
+# The below variable will contain the classpath for all the jar dependencies.
+jars=$(ls extracted/*/*/*.jar | tr \\n :)
+
+while true
+do
+  nc -v -l -p 9220 |  java -cp build/libs/KafkaPrinter.jar:"$jars" org.opensearch.KafkaPrinter -o "$target_endpoint" "$topic_name"
```

Comment:
Netcat is not piping in here but rather the std out of the jar should be piped to netcat and thus to the Replayer (which should be 9210), similar to:
`java -cp build/libs/KafkaPrinter.jar:"$jars" org.opensearch.KafkaPrinter -o "$bootstrap_server" "$topic_name" | nc -v localhost 9210`